interface-definitions/vpn_sstp.xml.in

```diff
@@ -53,6 +53,15 @@
           #include <include/accel-ppp/wins-server.xml.i>
           #include <include/generic-description.xml.i>
           #include <include/name-server-ipv4-ipv6.xml.i>
+          <leafNode name="host-name">
+            <properties>
+              <help>Allow SSTP connection to specified host and with the same TLS SNI</help>
```

Comment:
Can we change the help to: `Only allow connection to specified host with the same TLS SNI` please

Reply:
Done